tutorials/fundamentals/async-foreach.md

```diff
@@ -0,0 +1,15 @@
+You should **not** make the callback function parameter in `forEach()`
+an async function as you cannot guarantee order of iteration.
+[Here's More Info Why.](https://thecodebarbarian.com/basic-functional-programming-with-async-await.html)
+If you wish to still pursue that path, here is a better alternative:
+
+```javascript
+async function test() {
```

Comment:
This is an example of executing async functions in parallel. I'd also demonstrate executing them in series using `for/of`.